### PR TITLE
Bump release for 6.0.0

### PR DESCRIPTION
**Pull request by petems**
* Updates CHANGELOG.md

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-splunk",
-  "version": "5.1.1-rc0",
+  "version": "5.2.0",
```
Comment (bastelfreak) [inline metadata.json]: correct PR title but wrong version? :D

Comment (petems) [inline metadata.json]: D'oh! 😓

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,115 +1,187 @@
...
+Each new release typically also includes the latest modulesync defaults.
+These should not impact the functionality of the module.
 
-This is the last release with Pupept 3 support!
-* Modulesync with latest Vox Pupuli defaults
-* Fix several markdown issues
-* Add missing badges
-* Fix several rubocop issues
-* Bump min version_requirement for Puppet + deps
+## [v5.2.0](https://github.com/voxpupuli/puppet-splunk/tree/v5.2.0) (2017-05-25)
```
Comment (bastelfreak) [inline CHANGELOG.md]: please adjust this as well

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,115 +1,187 @@
...
+[Full Changelog](https://github.com/voxpupuli/puppet-splunk/compare/v5.1.0...v5.2.0)
```
Comment (bastelfreak) [inline CHANGELOG.md]: and this

File: CHANGELOG.md (hunk 2 above)
Comment (petems) [inline CHANGELOG.md]: Done! 👍

File: CHANGELOG.md (hunk 3 above)
Comment (petems) [inline CHANGELOG.md]: Done! 👍

**Pull request merged**